lib/evm_database_ops.rb

```diff
@@ -116,13 +118,15 @@ def self.restore(db_opts, connect_opts = {})
       end
 
       session = MiqGenericMountSession.new_session(connect_opts)
-      db_opts[:local_file] = session.uri_to_local_path(uri)
+      db_opts[:local_file]  = session.uri_to_local_path(uri)
+      db_opts[:object_file] = session.uri_to_object_path(uri) if connect_opts[:object_store]
```

Comment:
If you're using the session object to get this value, why pass it in separately to `#add` above? Couldn't the session object internally make the same call?

Reply:
Related: https://github.com/ManageIQ/manageiq-gems-pending/pull/355#discussion_r202809442